platform/podjs/src/manifest-parser/src/manifest.ts

```diff
@@ -5,7 +5,6 @@ import { parse as parseSemVer, SemVer, Range } from "semver";
 
 export interface MainManifest {
     readonly name: string;
```

Comment:
In this case, the path includes `src` twice. Rearranging a bit the directories might make it a bit simpler.